kafka-ui-react-app/src/components/common/JSONViewer/StyledWrapper.styled.ts

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export const StyledWrapper = styled.div`
   background-color: #f9fafa;
   padding: 8px 16px;
-  max-height: 516px;
-  overflow-y: scroll;
+  max-height: 575px;
+  overflow-y: hidden;
```

Comment:
if we add overflow hidden, some schemas versions are not completely seen and aren't scrollable either

Reply:
Problem is that we should have one scroll but it appeared to have two scrolls.


<img width="227" alt="Screen Shot 2022-01-17 at 5 43 52 PM" src="https://user-images.githubusercontent.com/96067981/149779751-61e00f6f-cc5e-4b75-a640-68bd155cecd8.png">

Reply:
It seems like we can control the height of `JSONViewer` from settings in `src/components/common/JSONViewer/JSONViewer.tsx`

`maxLines` option

Reply:
I've managed to get rid of 
```css
max-height: 575px;
overflow-y: hidden;
overflow-wrap: anywhere;
```
By setting `maxLines` to 28  (which is 548px in height)

I think if we add optional prop `maxLines` to `JSONViewer` and use this prop in places where we need smaller/bigger json viewers it will be not that bad

Reply:
Okay, good suggestion, thanks, I'll change.